lldb/test/API/tools/lldb-dap/stackTraceMissingFunctionName/TestDAP_stackTraceMissingFunctionName.py

```diff
@@ -8,6 +8,8 @@
 
 class TestDAP_stackTraceMissingFunctionName(lldbdap_testcase.DAPTestCaseBase):
     @skipIfWindows
+    # Jumping to address 0 will fail PAC signing before crashign on a bad frame.
```

Comment:
```suggestion
    # Jumping to address 0 will fail PAC signing before crashing on a bad frame.
```

Reply:
I'll fix this in 21.x & next if that's ok, to avoid having to do a rerun of CI. (unless we find more things to change)